ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1413,6 +1442,14 @@ export const WorkspaceEdit = fp.flow(
                 'Other methods of disseminating research findings'
               )
             : {},
+        aianResearchType: this.getAskAboutAIAN()
+          ? {
+              presence: true,
+            }
+          : {},
+        aianResearchDetails: this.getAskAboutAIAN()
+          ? requiredStringWithMaxLength(1000)
```

Comment:
this isn't working correctly for me: chars over 1000 show the generic missing message, not too-many-chars

Reply:
The issue you mentioned on line 2516 addresses this. Thank you!